index.test.js

```diff
@@ -0,0 +1,205 @@
+/* eslint-env jest */
```

Comment:
Had to add this comment to get `standard` to not complain about `jest` globals like `it`, `beforeAll`, etc.